### PR TITLE
us2

### PR DESCRIPTION
**Pull request by KevinBlbd**

### CODE REVIEW

File: components/Form/index.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+export default function Form({ formName, defaultData, onAddPiece }) {
+  const router = useRouter();
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    onAddPiece(data);
+    // router.push("/");
```
Comment (botacode-gh) [inline components/Form/index.js:38]: Commented out code

File: components/Form/index.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      />
+      <Label htmlFor="name">Artist name</Label>
+      <Input
+        id="artist name"
+        name="name"
+        type="text"
+        defaultValue={defaultData?.name}
+        required
+      />
+      {/* <Label htmlFor="map-url">Map Url</Label>
```
Comment (botacode-gh) [inline components/Form/index.js:68]: Here too

File: pages/index.js [hunk 3]
```diff
@@ -1,33 +1,49 @@
...
+        <StylePopup>
+          <Form onAddPiece={handleAddPiece} onClick={handleModalIsVisible} />
+          {""}
+        </StylePopup>
+      )}
     </div>
   );
 }
+
+// onCloseModal={handleCloseModal}
```
Comment (botacode-gh) [inline pages/index.js:49]: ici aussi ;)
```suggestion
```

**Review by botacode-gh — APPROVED** [review]
Few commented out code I think you can remove, otherwise LGTM!

File: pages/index.js (hunk 3 above)
Comment (KevinBlbd) [reply pages/index.js:49]: merci Xav ! 🦎

**Pull request merged**